pkg/apis/sources/v1/sinkbinding_lifecycle.go

```diff
@@ -94,6 +94,15 @@ func (sbs *SinkBindingStatus) MarkSink(uri *apis.URL) {
 	}
 }
 
+func (sbs *SinkBindingStatus) MarkSinkCACerts(certs *string) {
+	sbs.SinkCACerts = certs
+	if certs != nil {
+		sbCondSet.Manage(sbs).MarkTrue(SinkBindingConditionSinkCACertsProvided)
+	} else {
+		sbCondSet.Manage(sbs).MarkFalse(SinkBindingConditionSinkCACertsProvided, "SinkCACertsEmpty", "Sink CA Certs has resolved to empty.%s", "")
+	}
+}
+
 // Do implements psbinding.Bindable
 func (sb *SinkBinding) Do(ctx context.Context, ps *duckv1.WithPod) {
```

Comment:
We need to add some tests for the changes in this file here: https://github.com/knative/eventing/blob/main/pkg/apis/sources/v1/sinkbinding_lifecycle_test.go